web/components/Nav/Nav.tsx

```diff
@@ -100,7 +104,7 @@ export const Nav = ({ onFrontPage, currentPath, ticketsUrl }: NavProps) => {
             <BasicMenuItem urlPath="/about" label="About" />
           </ul>
           <div className={styles.ticketButton}>
-            <ButtonLink url={ticketsUrl} text="Tickets" />
+            <ButtonLink url={ticketsUrl} text="Tickets" openInNewTab={true} />
```

Comment:
```suggestion
            <ButtonLink url={ticketsUrl} text="Tickets" openInNewTab />
```

(Nit.)

Reply:
Thanks, I sometimes forget some of these "uncanny valley" JSX-vs-HTML things.

I found this note in the ["JSX in depth" docs](https://reactjs.org/docs/jsx-in-depth.html#props-default-to-true), though: "In general, we don’t recommend _not_ passing a value for a prop, because it can be confused with the [ES6 object shorthand](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015) `{foo}` which is short for `{foo: foo}` rather than `{foo: true}`. This behavior is just there so that it matches the behavior of HTML."

Still prefer the "value-less" syntax? I don't really have any opinions on it myself.

Reply:
TIL! :eyes: I was just pushing my personal preference (hence the nit :stuck_out_tongue:) but if this is _against_ what's recommended, I'll have to change my ways. :sweat_smile: